tests/test_bar_data.py

```diff
@@ -0,0 +1,418 @@
+from unittest import TestCase
+
+from testfixtures import TempDirectory
+import pandas as pd
+import numpy as np
+
+from zipline.data.data_portal import DataPortal
+from zipline.data.minute_bars import BcolzMinuteBarWriter, \
+    US_EQUITIES_MINUTES_PER_DAY, BcolzMinuteBarReader
+from zipline.data.us_equity_pricing import BcolzDailyBarReader
+from zipline.finance.trading import TradingEnvironment
+from zipline.protocol import BarData
+from zipline.utils.test_utils import write_minute_data_for_asset, \
+    create_daily_df_for_asset, DailyBarWriterFromDataFrames, assert_same
+
+OHLC = ["open", "high", "low", "close"]
+OHLCP = OHLC + ["price"]
+ALL_FIELDS = OHLCP + ["volume", "last_traded"]
+
+# offsets used in test data
+field_info = {
+    "open": 1,
```

Comment:
Could we make these mantissas instead?
So that same value is not in different slots for OHLC. e.g. low of i, and close of i+1 would have the same value.